linkml/generators/sqltablegen.py

```diff
@@ -159,25 +172,20 @@ def dump(sql, *multiparams, **params):
         sqltr = RelationalModelTransformer(SchemaView(self.schema))
         if not self.use_foreign_keys:
             sqltr.foreign_key_policy = ForeignKeyPolicy.NO_FOREIGN_KEYS
-        tr_result = sqltr.transform(
-            tgt_schema_name=kwargs.get("tgt_schema_name", None), top_class=kwargs.get("top_class", None)
-        )
+        tr_result = sqltr.transform(tgt_schema_name=kwargs.get("tgt_schema_name"), top_class=kwargs.get("top_class"))
```

Comment:
`object.get("property")` returns `None` by default, so no need to specify it as a fallback